dali/pipeline/pipeline.h

```diff
@@ -348,6 +375,23 @@ class DLL_PUBLIC Pipeline {
    */
   DLL_PUBLIC void Outputs(Workspace *ws);
 
+
+  DLL_PUBLIC std::vector<Horcrux> GetHorcruxes() {
+    std::vector<Horcrux> ret;
+    for (auto so : stream_operators_) {
+      auto h = so->GetHorcruxesBack();
```

Comment:
How much work would it be to propagate the horcrux through the executor than get directly from the operator (to enable pipeline)?